AquaMQTT/src/main.cpp

```diff
@@ -20,51 +24,88 @@ MQTTTask       mqttTask;
 OTAHandler     otaHandler;
 RTCHandler     rtcHandler;
 WifiHandler    wifiHandler;
+WebHandler     webHandler;
 
 esp_task_wdt_config_t twdt_config = {
     .timeout_ms     = WATCHDOG_TIMEOUT_MS,
     .idle_core_mask = (1 << configNUM_CORES) - 1,
     .trigger_panic  = true,
 };
+bool                                   configOK = false;
+extern struct aquamqtt::AquaMqttStruct aquamqttSettings;
 
 void loop()
 {
     // watchdog
     esp_task_wdt_reset();
     delay(1);
 
-    // handle wifi events
     wifiHandler.loop();
 
     // handle over-the-air module in main thread
     otaHandler.loop();
 
     // handle real-time-clock module in main thread
     rtcHandler.loop();
+
+    webHandler.loop();
 }
 
 void setup()
 {
     // limited serial output for debuggability
     Serial.begin(9600);
     Serial.println("REBOOT");
+    if (!LittleFS.begin())
+    {
+        Serial.println("Cannot open FileSystem");
+        return;
+    }
+    // // Try to load config
+    if (!loadWifiConfig())
+    {
+        Serial.println("Wifi conf not loaded ....");
+    }
+    else
+    {
+        configOK = true;
+    }
+
+    if (configOK)
+    {
+        if (!wifiHandler.setup())
+        {
+            wifiHandler.setupAP();
+        }
+    }
+    else
+    {
+        Serial.println("Wifi set to Access Point Mode");
+        wifiHandler.setupAP();
+    }
```

Comment:
```suggestion
    if (!loadWifiConfig())
    {
        Serial.println("Wifi conf not loaded ....");
         Serial.println("Wifi set to Access Point Mode");
        wifiHandler.setupAP();
    }
    else
    {
        if (!wifiHandler.setup())
        {
            wifiHandler.setupAP();
        }
    }

```

Do you really need the `configOK` field?